### PR TITLE
Users/ersciple/m95locstrings

### PR DESCRIPTION
**Pull request by ericsciple**
Is the fix already in master?  If not, link the bug tracking that work.
No. I've added a linked work item to the bug.

What testing was done?
Tested E2E from an agent on a Linux box connected to a German TFS SKU.

Does this fix change the AT/DT contract?
No.

Does this fix include any database servicing changes?
No. But we might need a new servicing change to get the new tasks into TFS (if one hasn't already been created for the patch milestone). I'll open a bug and follow-up with CAT team.

Were tests added to cover this to make sure it doesn't regress?  If not, link a bug or user story tracking that work.
Yes.

### CODE REVIEW

**Comment by scottdallamura** [comment]
looks good

**Comment by TingluoHuang** [comment]
Looks good.

File: node/lib/task.ts [hunk 1]
```diff
@@ -153,14 +153,24 @@ function loadLocStrings(resourceFile: string, culture: string): { [key: string]:
     if (exist(resourceFile)) {
         debug('load strings from: ' + resourceFile);
         var resourceJson = require(resourceFile);
```
Comment (ericsciple) [inline node/lib/task.ts]: The changes to this file fix the case-sensitivity issue on xplat build agents.

File: node/package.json [hunk 2]
```diff
@@ -38,6 +38,6 @@
     "merge2": "^0.3.6",
     "mocha": "^2.2.5",
     "semver": "^5.1.0",
-    "typescript": "next"
+    "typescript": "1.8.0-dev.20160104"
```
Comment (ericsciple) [inline node/package.json]: This is a developer tools dependency only. This pins the CI server to a known tsc version that won't break our build.

**Pull request merged**